### PR TITLE
Runnable examples: 40 total, added 9

### PR DESCRIPTION
**Pull request by rasmus-kirk**
## Purpose

See #153 

## Changes

Added examples:

 - `getBlockItemStatus`
 - `getNodeInfo`
 - `getPassiveDelegationInfo`
 - `getPassiveDelegators`
 - `getPassiveDelegatorsRewardPeriod`
 - `getPeersInfo`
 - `getPoolDelegators`
 - `getPoolDelegatorsRewardPeriod`
 - `getPoolInfo`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -19,6 +19,6 @@
   "scripts": {
     "lint": "eslint . --cache --ext .ts,.tsx --max-warnings 0",
     "lint-fix": "yarn lint --fix; exit 0",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit && echo 'Typehchecked successfully'"
```
Comment (shjortConcordium) [inline examples/package.json:22]:
```suggestion
    "typecheck": "tsc --noEmit && echo 'Typechecked successfully'"
```
Why do you need to echo the statement? :thinking:

File: examples/client/getPassiveDelegationInfo.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Retrieves information about the passive delegators, at the end of the specified block.
+(async () => {
+    const bakerPoolInfo: PassiveDelegationStatus =
+        await client.getPassiveDelegationInfo(cli.flags.block);
+
+    console.dir(bakerPoolInfo, { depth: null, colors: true });
```
Comment (shjortConcordium) [inline examples/client/getPassiveDelegationInfo.ts:51]:
```suggestion
    const passiveDelegationInfo: PassiveDelegationStatus =
        await client.getPassiveDelegationInfo(cli.flags.block);

    console.dir(passiveDelegationInfo, { depth: null, colors: true });
```

Also missing something like this: :upside_down_face: 
`const peerVersion: string = nodeInfo.peerVersion;`

File: examples/client/getPassiveDelegationInfo.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    Number(port),
+    credentials.createInsecure(),
+    { timeout: 15000 }
+);
+
+if (cli.flags.h) {
+    cli.showHelp();
+}
+
+/// Retrieves information about the passive delegators, at the end of the specified block.
```
Comment (shjortConcordium) [inline examples/client/getPassiveDelegationInfo.ts:46]:
```suggestion
/// Retrieves information about the passive delegation, including the total capital delegation and the current commission rates for passive delegation, at the end of the specified block.
```
Maybe :thinking:

File: examples/client/getNodeInfo.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    --endpoint, -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            account: {
+                type: 'string',
+                alias: 'a',
+                isRequired: true,
+            },
```
Comment (shjortConcordium) [inline examples/client/getNodeInfo.ts:24]:
```suggestion
```

File: examples/client/getNodeInfo.ts [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+/// Get information about the node.
+
+/// The `NodeInfo` includes information of:
+///  - Meta information, such as the, version of the node, type of the node, uptime
+///    and the local time of the node.
+///  - NetworkInfo, which yields data such as the node id, packets sent/received,
+///    average bytes per second sent/received.
+///  - ConsensusInfo. The `ConsensusInfo` returned depends on if the node supports
```
Comment (shjortConcordium) [inline examples/client/getNodeInfo.ts:58]:
```suggestion
///  - ConsensusStatus. The `ConsensusStatus` returned depends on if the node supports
```
I can only find a status, when I try. :thinking:

File: examples/package.json (hunk 1 above)
Comment (rasmus-kirk) [reply examples/package.json:22]: Just to indicate that something is happening, we can remove it if you like, I don't feel strongly on i

File: examples/client/getNodeInfo.ts (hunk 5 above)
Comment (rasmus-kirk) [reply examples/client/getNodeInfo.ts:58]: They are the same right? It's just the V2 name that we decided not to use. Still better to keep it consistent with `ConsensusStatus` in the docs as well

**Review by soerenbf — APPROVED** [review]
Generally looks good.

Where did the idea for the colors in terminal output come from? I like the output, but it kind of obscures the code a little, so I don't know what we're focusing on here: cleaner code or better output?

**Comment by abizjak** [comment]
@rasmus-kirk please don't add needless complexity as @soerenbf stated. Coloring output is needless complexity for these examples. They should focus on how to use the SDK, not how to pretty print stuff.

File: examples/client/getPassiveDelegationInfo.ts [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+ * Retrieves information about the passive delegation, including the total
+ * capital delegation and the current commission rates for passive delegation,
+ * at the end of the specified block.
+ */
+
+(async () => {
+    const passiveDelegationInfo: PassiveDelegationStatus =
+        await client.getPassiveDelegationInfo(cli.flags.block);
+
+    //console.dir(passiveDelegationInfo, { depth: null, colors: true });
```
Comment (shjortConcordium) [inline examples/client/getPassiveDelegationInfo.ts:53]:
```suggestion
```
:wastebasket:

File: examples/package.json (hunk 1 above)
Comment (shjortConcordium) [reply examples/package.json]: No I guess it okay, as long as it doesn't make it less usable in the CI (but we'll look at that when we get there)

**Review by shjortConcordium — APPROVED** [review]
I like the colours. :shrug:

File: examples/package.json (hunk 1 above)
Comment (rasmus-kirk) [reply examples/package.json]: I'll remove it if it's a problem then :+1:

**Pull request merged**